Extension/lib/filter/rules/url-filter-rule.js

```diff
@@ -646,7 +657,6 @@
     UrlFilterRule.contentTypes.ALL |= UrlFilterRule.contentTypes.FONT;
     UrlFilterRule.contentTypes.ALL |= UrlFilterRule.contentTypes.WEBSOCKET;

```

Comment:
Why not? `WEBSOCKET` is a request type, just like `IMAGE` or `SCRIPT`.
`WEBRTC` isn't supported by browsers now, but it's possible in the future.